apps/ssr-tests-v9/src/utils/generateEntryPoints.ts

```diff
@@ -0,0 +1,65 @@
+// @ts-expect-error There are no typings for this module
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { getVnextStories } from '@fluentui/public-docsite-v9/.storybook/main.utils';
+import * as glob from 'glob';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as prettier from 'prettier';
+
+import { getImportsFromIndexFile } from './getImportsFromIndexFile';
+
+type GenerateEntryPointsConfig = { cjsEntryPoint: string; esmEntryPoint: string };
+
+export async function generateEntryPoints(config: GenerateEntryPointsConfig): Promise<void> {
+  const rawStoriesGlobs = getVnextStories() as string[];
+  const storiesGlobs = rawStoriesGlobs
+    // TODO: Find a better way for this. Pass the path via params? 👇
+    .map(pattern => path.resolve(__dirname, '..', pattern));
```

Comment:
I want to reuse the same globs, but we operate from a different `cwd` there. Opened for better ideas.